examples/headsail-bsp/src/mmap.rs

```diff
@@ -1 +1,2 @@
 pub(crate) const UART0_ADDR: usize = 0xFFF00000;
+pub(crate) const TIMER0_ADDR: usize = 0x5_0000;
```

Comment:
Thanks for adding the underscore here. I think it's generally a good idea to include those in the addresses.